taccsite_cms/static/djangocms_forms/css/djangocms_forms.css

```diff
@@ -0,0 +1,3 @@
+/* SEE: ../../site_cms/css/src/app.djangocms_forms.css */
+/* FAQ: We uses future syntax that must be processed to work in the present;
+        and the process does not support outputting files to this directory */
```

Comment:
<details><summary>But the process could support it, one day. I have goals.</summary>

The process may be able to support this. I would rather put the CSS here. I could put most of it here, but then I cannot declaratively or sensibly explain that more code for it is in static. I hope to re-visit the CMS style build process. I think I may even need to for some core-styles to do that I have forgotten at the moment.

</details>